webapp2.py

```diff
@@ -211,6 +213,39 @@ def get(self, argument_name, default_value='', allow_multiple=False):
         else:
             return default_value
 
+    def get(self, argument_name, default_value='', allow_multiple=False):
+        """Returns the query or POST argument with the given name.
+
+        We parse the query string and POST payload lazily, so this will be a
+        slower operation on the first call.
+
+        :param argument_name:
+            The name of the query or POST argument.
+        :param default_value:
+            The value to return if the given argument is not present.
+        :param allow_multiple:
+            Return a list of values with the given name (deprecated).
+        :returns:
+            Return the value with the given name given in the request. If there
+            are multiple values, this will only return the first one. Use
+            `get_all()` to get multiple values.
+        """
+        return self._get(argument_name,
```

Comment:
Sounds like a silly Java rule. :P 

In this case, `__getitem__` is a *de jure* alias to `get`, so I'm fine with the latter calling the former.